conda/base/context.py

```diff
@@ -355,6 +356,16 @@ class Context(Configuration):
     unsatisfiable_hints = ParameterLoader(PrimitiveParameter(True))
     unsatisfiable_hints_check_depth = ParameterLoader(PrimitiveParameter(2))
 
+    prerelease_behavior = ParameterLoader(PrimitiveParameter(PrereleaseBehavior.ALLOW,
+                                                             element_type=PrereleaseBehavior))
+    prerelease_allowed_packages = ParameterLoader(
+        SequenceParameter(
+            PrimitiveParameter("", element_type=str),
+            default=("openssl <=2.99.99",),
+            string_delimiter='&',
+        )
+    )
+
```

Comment:
I don't think we need to expose both the raw string _and_ a spec. This should be stored as `_prerelease_allowed_packages`, with a Parameter alias `prerelease_allowed_packages`. Then, the property below can just be `prerelease_allowed_packages`. You can see this pattern in other property-post-processed keys.